Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -267,18 +268,21 @@ def com_google_fonts_check_description_broken_links(description_html):
                     f"Found an email address: {link}")
       continue
 
-    try:
-      response = requests.head(link, allow_redirects=True, timeout=10)
-      code = response.status_code
-      if code != requests.codes.ok:
-        broken_links.append(f"{link} (status code: {code})")
-    except requests.exceptions.Timeout:
-      yield WARN,\
-            Message("timeout",
-                    f"Timedout while attempting to access: '{link}'."
-                    f" Please verify if that's a broken link.")
-    except requests.exceptions.RequestException:
-      broken_links.append(link)
+    # avoid requesting the same URL more then once
+    if link not in unique_links:
```

Comment:
no matter if set or list, but this is using the `my_set.add()` api:

```py
if link in unique_links:
  continue
unique_links.add(link)
```

Will keep the indentation level lower.

Reply:
Will also make the diff smaller.

Reply:
cool! I will also check this before yielding `INFO "email"` so that we won't have duplicate reporting in there as well